ApiBundle/Controller/DeletedController.php

```diff
@@ -16,7 +18,11 @@
  */
 class DeletedController extends BaseController
```

Comment:
Would not it be more accurate to name this controller TrashcanController ? It seems used to display trashitems, isn't it ?